Specs/DateTime/Swedish/TimeZoneParser.json

```diff
@@ -0,0 +1,1048 @@
+[
+  {
+    "Input": "Boka ett rum åt mig Amazonsk sommartid",
+    "Debug": true,
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupportedByDesign": "javascript, python, java",
+    "Results": [
+      {
+        "Text": "amazonsk sommartid",
+        "Type": "timezone",
+        "Value": {
+          "TimeZoneResolution": {
+            "Value": "UTC+04:30",
+            "UtcOffsetMins": 270
+          }
+        },
+        "Start": 20,
+        "Length": 18
+      }
+    ]
+  },
+  {
+    "Input": "Book me a room at beijing time",
```

Comment:
Just translation of these specs and adding the ambiguous resolution values in the dict and we're good to merge. 
Please add a couple new sentences that use Swedish-specific abbreviations too, including ambiguous ones, to document the behaviour in those cases.
Almost there. Thanks!

Reply:
Will do. The biggest issue right now is the "you don't need to add 'time' or 'time zone'" to the entries in the FullToMinMapping dictionary. I need to sort that out first, since it currently prevents the parser from finding the entry in the dictionary resulting in "UTC+xx:xx..." resolutions. It isn't just a matter of adding "tid" to the regex... But, I will find a solution.

Reply:
Hi @tellarin! I think I have found the reason for the parser not being able to get the correct timezone offset. It turns out that the parser calls `BaseTimeZoneParser` which is not localized, i.e. it doesn't use the swedish version of `TimeZoneDefinitions` when looking up the TZ in the `FullToMinMapping` dictionary of that class. As I see it we need to update `BaseTimeZoneParser` to be localizeable just as the corresponding `BaseTimeZoneExtractor` class.

Reply:
I have added a draft PR #2832 as a suggestion of how to solve this.

Reply:
@pete1854 Merged. Thanks a lot!